src/userbase-server/app.js

```diff
@@ -526,6 +575,62 @@ exports.listUsersWithPagination = async function (req, res) {
   }
 }
 
+exports.listUsersForDatabaseWithPagination = async function (req, res) {
```

Comment:
Minor nit: figure this would make a bit more sense to export in the `db.js` file

Reply:
I could do that. One consequence is that `_validateAppId`, `_validateListUsersLastEvaluatedKey`, and `_validateAppResponseToGetApp` would have to either be copied verbatim from app.js to db.js, or they would have to be exported. What do you think?

Reply:
See other newly added comment, think you're going to need a new function `_validateListDatabaseUsersLastEvaluatedKey` anyway. Fine to export `_validateAppId` and `_validateAppResponseToGetApp`

Reply:
Done!